packages/ui/src/AdminAddMCQ.tsx

```diff
@@ -0,0 +1,137 @@
+import { useEffect, useState } from "react";
+import { Sheet, SheetContent, SheetTrigger } from "./shad/ui/sheet";
+import { Button } from "./shad/ui/button";
+import { Input } from "./shad/ui/input";
+import { Card, CardDescription, CardTitle } from "./shad/ui/card";
+import { MCQQuestion, Problem } from "@prisma/client";
+import { createMCQ, deleteMCQ } from "../../../apps/web/components/utils";
+
+interface AdminAddMCQProps extends Problem {
+  mcqQuestions: MCQQuestion[];
+}
+
+const AdminAddMCQ = ({ problem }: { problem: AdminAddMCQProps }) => {
+  const [mcqs, setMcqs] = useState<MCQQuestion[]>([]);
+  const [ExistingMCQs, setExistingMCQs] = useState<MCQQuestion[]>(problem.mcqQuestions);
```

Comment:
Use camel case for variables.The name ExistingMCQs starts with an uppercase letter, which in React usually signifies a component.